base/setup/usetup/native/utils/console.c

```diff
@@ -159,9 +159,10 @@ FlushConsoleInputBuffer(
     KEYBOARD_INPUT_DATA InputData;
     NTSTATUS Status;
 
+    Offset.QuadPart = 0;
+    Timeout.QuadPart = -100;
```

Comment:
These are not improvements. Having these numbers assigned close to where they are used makes more sense.

Reply:
Reverted.